src/node_errors.h

```diff
@@ -68,11 +68,14 @@ void AppendExceptionLine(Environment* env,
   V(ERR_INVALID_ARG_VALUE, TypeError)                                          \
   V(ERR_OSSL_EVP_INVALID_DIGEST, Error)                                        \
   V(ERR_INVALID_ARG_TYPE, TypeError)                                           \
+  V(ERR_INVALID_FILE_URL_HOST, TypeError)                                      \
+  V(ERR_INVALID_FILE_URL_PATH, TypeError)                                      \
   V(ERR_INVALID_OBJECT_DEFINE_PROPERTY, TypeError)                             \
   V(ERR_INVALID_MODULE, Error)                                                 \
   V(ERR_INVALID_STATE, Error)                                                  \
   V(ERR_INVALID_THIS, TypeError)                                               \
   V(ERR_INVALID_TRANSFER_OBJECT, TypeError)                                    \
+  V(ERR_INVALID_URL_SCHEME, TypeError)                                         \
```

Comment:
url has protocol. i think this needs to be invalid_url_protocol

Reply:
I just follow the original implementation:

https://github.com/nodejs/node/blob/cd6bd3f0fb135d3ef7cc847d62205c6f0d7a47c8/lib/internal/url.js#L1359-L1360C29

Should I change the error message?